<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.oribo.android365.testcase.SignIn" tests="4" time="499.748" ignored="9" failures="1" timestamp="17 三月 2017 09:05:35 GMT" skipped="1" hostname="test" errors="0">
  <testcase name="b_logff" time="0.000" classname="com.oribo.android365.testcase.SignIn">
    <skipped/>
  </testcase> <!-- b_logff -->
  <testcase name="a_phoneregister" time="382.636" classname="com.oribo.android365.testcase.SignIn">
    <failure type="java.lang.AssertionError" message="60s内未收到验证码 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 60s内未收到验证码 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.oribo.android365.testcase.SignIn.a_phoneregister(SignIn.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- a_phoneregister -->
  <testcase name="weibologging" time="64.889" classname="com.oribo.android365.testcase.SignIn"/>
  <testcase name="checkother" time="52.223" classname="com.oribo.android365.testcase.SignIn"/>
  <testcase name="checkbind" classname="com.oribo.android365.testcase.Personcenter">
    <skipped/>
  </testcase> <!-- checkbind -->
  <testcase name="checkdefaultphoto" classname="com.oribo.android365.testcase.Personcenter">
    <skipped/>
  </testcase> <!-- checkdefaultphoto -->
  <testcase name="modifypassword" classname="com.oribo.android365.testcase.Personcenter">
    <skipped/>
  </testcase> <!-- modifypassword -->
  <testcase name="authorization" classname="com.oribo.android365.testcase.Personcenter">
    <skipped/>
  </testcase> <!-- authorization -->
  <testcase name="delectscene" classname="com.oribo.android365.testcase.HomeManage">
    <skipped/>
  </testcase> <!-- delectscene -->
  <testcase name="ClickMore" classname="com.oribo.android365.testcase.HomeManage">
    <skipped/>
  </testcase> <!-- ClickMore -->
  <testcase name="addscene" classname="com.oribo.android365.testcase.HomeManage">
    <skipped/>
  </testcase> <!-- addscene -->
  <testcase name="editscene" classname="com.oribo.android365.testcase.HomeManage">
    <skipped/>
  </testcase> <!-- editscene -->
  <testcase name="checkscene" classname="com.oribo.android365.testcase.HomeManage">
    <skipped/>
  </testcase> <!-- checkscene -->
</testsuite> <!-- com.oribo.android365.testcase.SignIn -->
